packages/@sfx/sayt-driver-plugin/test/unit/browser/sayt-driver-plugin.test.ts

```diff
@@ -0,0 +1,193 @@
+import { expect, spy, stub } from '../../utils';
+import { SaytDriverPlugin } from '../../../src/index';
+import { DomEventsPlugin } from '../../../../dom-events-plugin/src/index';
+import { SaytPlugin } from '../../../../sayt-plugin/src';
+
+describe('Sayt Driver Plugin', () => {
+  let Driver: any;
+
+  beforeEach(() => {
+    Driver = new SaytDriverPlugin();
+    Driver.register({
```

Comment:
This should not be relying on `register` actually working. Each test should instead inject an object into `driver.core` that contains the expected plugins by setting the `core` property directly.